cfme/storage/volume.py

```diff
@@ -105,6 +105,7 @@ class VolumeAddForm(View):
     volume_name = TextInput(name='name')
     size = TextInput(name='size')
     tenant = BootstrapSelect(name='cloud_tenant_id')
+    volume_type = BootstrapSelect(name='volume_type')
```

Comment:
Optional:
Please order view elements if possible like...
```
storage_manager = BootstrapSelect(name='storage_manager_id')
tenant = BootstrapSelect(name='cloud_tenant_id')
volume_name = TextInput(name='name')
volume_type = BootstrapSelect(name='volume_type')
size = TextInput(name='size')
```
It will help in collective fill.

Reply:
Done

Reply:
awesome thanks :+1: